General queries/psexec elevation to SYSTEM.txt

```diff
@@ -0,0 +1,5 @@
+ProcessCreationEvents 
+| where EventTime > ago(7d) 
+| where FileName == "psexec.exe"
+| where ProcessCommandLine contains "-s "
```

Comment:
you should also add cases where users used /s instead of -s

Reply:
I will add it now